packages/desktop-client/src/components/modals/ImportTransactions.js

```diff
@@ -584,6 +584,8 @@ export default function ImportTransactions({ modalProps, options }) {
 
   let [parseDateFormat, setParseDateFormat] = useState(null);
 
+  let [clearOnImport, setClearOnImport] = useState(true);
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏could we instead set this to `false` by default? I wouldn't want to break the mental model existing users have when doing manual imports.
```suggestion
  let [clearOnImport, setClearOnImport] = useState(false);
```

Reply:
Thanks for reviewing. I believe the default behaviour at the moment on import is to mark the transactions as cleared

Reply:
Yes, you're actually correct. Thanks for correcting me!